WordPress/Classes/Utility/WebViewController/WPWebViewController.m

```diff
@@ -177,6 +177,7 @@ - (void)setupTitle
 
 - (void)setupWebView
 {
+    self.webView.customUserAgent = [WPUserAgent wordPressUserAgent];
```

Comment:
This old `WPWebViewController` is only used by the social connection screen, so nothing else needs to be tested.